src/python/nimbusml/preprocessing/schema/prefixcolumnconcatenator.py

```diff
@@ -17,38 +18,15 @@
 from ...internal.utils.utils import trace
 
 
-class PrefixColumnConcatenator(core, BaseTransform, TransformerMixin):
+class PrefixColumnConcatenator(
+        core,
+        BaseTransform,
+        TransformerMixin):
     """
 
-    Combines several columns into a single vector-valued column by prefix.
+    Concatenates one or more columns of the same item type.
 
-    .. remarks::
-        ``PrefixColumnConcatenator`` creates a single vector-valued column from
-        multiple
-        columns. It can be performed on data before training a model. The
-        concatenation
-        can significantly speed up the processing of data when the number of
-        columns
-        is as large as hundreds to thousands.
-
-    :param columns: a dictionary of key-value pairs, where key is the output
-        column name and value is a list of input column names.
-
-         * Only one key-value pair is allowed.
-         * Input column type: numeric or string.
-         * Output column type:
-        `Vector Type </nimbusml/concepts/types#vectortype-column>`_.
-
-        The << operator can be used to set this value (see
-        `Column Operator </nimbusml/concepts/columns>`_)
-
-        For example
-         * ColumnConcatenator(columns={'features': ['age', 'parity',
-        'induced']})
-         * ColumnConcatenator() << {'features': ['age', 'parity',
-        'induced']})
-
-        For more details see `Columns </nimbusml/concepts/columns>`_.
```

Comment:
we need this doc

Reply:
Fixed in latest commit.

---
In reply to: [346961298](https://github.com/microsoft/NimbusML/pull/364#discussion_r346961298) [](ancestors = 346961298)